src/d3a/d3a_core/d3a_jobs.py

```diff
@@ -48,6 +48,13 @@ def start(scenario, settings):
     if advanced_settings is not None:
         update_advanced_settings(ast.literal_eval(advanced_settings))
 
+    spot_market_type = settings.get('spot_market_type', None)
+    if spot_market_type is not None:
+        if not 1 <= spot_market_type <= 3:
```

Comment:
This reminds me that we wanted to create a common interface between d3a and d3a-web. Shall we tackle this before moving forward and adding features? IMO yes. What do you think?

Reply:
I agree, but I think that it has to be done in parallel. One thing that comes to mind is that we need to create a separate 'common' repository that will keep this common interface. Nevertheless, I am opening [this task](https://gridsingularity.atlassian.net/browse/D3ASIM-1337) and creating corresponding subtasks in order to tackle this issue incrementally.